### PR TITLE
improvement: add guidance on how to run it on a VM

### PR DESCRIPTION
**Pull request by Shabirmean**
### Description
- The application is used for a [guide that shows how to run it in a VM inside ABM using Anthos VMRUntime](https://github.com/GoogleCloudPlatform/anthos-samples/tree/main/anthos-vmruntime)
- For the purpose of the guide we don't show how the VM was setup, we just provide the users with an image of a preconfigured VM that they can use. This is to speed up the process
- But it would be good to have the process of creating this VM documented so that in future anybody updating the guide knows to re-create it
- Also could help people who might want to run the app in a VM (rare case)

### Changes
- Add the script that can be run to set up the VM to run the application
- Add README about the script
- Add a line on the root README

### CODE REVIEW

File: extras/anthos-vmruntime/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+You can run all the services of the application in a single VM as a system
+service. Simply copy the [**init-vm.sh**](init-vm.sh) file into your VM and then
+execute it from inside the VM.
+```sh
+sudo bash init-vm.sh
+```
+
+This sets up all the services of the application to run as a system service
+inside your VM. Note that the `inventory service` and the `payments service`
```
Comment (ckim328) [inline extras/anthos-vmruntime/README.md:11]: nit, maybe format the : Note that the `inventory service`.... onto a newline

File: extras/anthos-vmruntime/init-vm.sh [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+cat <<EOF | sudo tee -a /etc/systemd/system/pos_apiserver.service -
+[Unit]
+Description=API Server of the Point of Sale Application
+PartOf=pos.service
+After=pos.service
+After=pos_inventory.service
+After=pos_payments.service
+
+[Service]
+User=shabirmean
```
Comment (ckim328) [inline extras/anthos-vmruntime/init-vm.sh:89]: is there a generic mock for a `User`?

Comment (Shabirmean) [reply extras/anthos-vmruntime/init-vm.sh]: Fixed in https://github.com/GoogleCloudPlatform/point-of-sale/pull/132/commits/ea19c176f9a1ca7a6ae80ff9e721de338c8b278b

**Review by ckim328 — DISMISSED** [review]
LGTM!

**Review by ckim328 — DISMISSED** [review]
LGTM!

**Comment by Shabirmean** [comment]
:zap:  Two deployments have been created for the Point-of-Sale application. You may access and test them at: :zap: </br>
- **MySQL DB** backed deployment: [**34.70.155.178**](http://34.70.155.178)
- **Embedded H2** DB backed deployment: [**34.68.187.57**](http://34.68.187.57)

**Comment by Shabirmean** [comment]
:zap:  Two deployments have been created for the Point-of-Sale application. You may access and test them at: :zap: </br>
- **MySQL DB** backed deployment: [**34.70.155.178**](http://34.70.155.178)
- **Embedded H2** DB backed deployment: [**34.68.187.57**](http://34.68.187.57)

**Comment by Shabirmean** [comment]
:zap:  Two deployments have been created for the Point-of-Sale application. You may access and test them at: :zap: </br>
- **MySQL DB** backed deployment: [**34.70.155.178**](http://34.70.155.178)
- **Embedded H2** DB backed deployment: [**34.68.187.57**](http://34.68.187.57)

**Comment by Shabirmean** [comment]
:zap:  Two deployments have been created for the Point-of-Sale application. You may access and test them at: :zap: </br>
- **MySQL DB** backed deployment: [**34.70.155.178**](http://34.70.155.178)
- **Embedded H2** DB backed deployment: [**34.68.187.57**](http://34.68.187.57)

**Pull request merged**